lib/blpapi-wrapper.ts

```diff
@@ -460,30 +465,36 @@ export class Session extends events.EventEmitter {
 
         log('Unsubscribing: ' + JSON.stringify(subscriptions));
 
-        var cids: number[] = [];
-        _.forEach(subscriptions, (s: Subscription, i: number): void => {
+        var caes = _.map(subscriptions, (s: Subscription): CidAndEvents => {
```

Comment:
Let's keep this as `number[]` and perform the look-up/unlisten of the events in the `_.forEach()` below?  It's less change to`unsubscribe` and the same end-result is achieved with no performance impact.